package.json

```diff
@@ -0,0 +1,28 @@
+{
+ "name": "anvil",
+ "version" : "0.1.0",  
+ "canonical" : "https://ncpi-fhir.github.io/ncpi-fhir-ig",
+ "url" : "https://ncpi-fhir.github.io/ncpi-fhir-ig",
+ "title" : "NCPI Implementation Guide",
+ "description": "The NIH Cloud-based Platform Interoperability (NCPI) FHIR Working Group Implementation Guide.",
```

Comment:
`"The NIH Cloud Platform Interoperability Effort (NCPI) FHIR Implementation Guide."`